### PR TITLE
 Integrating the DataTables JS library

### PR DESCRIPTION
**Pull request by alexiscott**

### CODE REVIEW

File: examples/customDataHandlers.js [hunk 1]
```diff
@@ -85,13 +85,51 @@ let customDataHandlers = {
...
 
+
+
+  // @TODO use data with dashboardData.
   getTableData: function (componentData, dashboardData, handler, e, appliedFilters, pipelineData) {
-    console.log(arguments);
-    if (dashboardData.climateData) {
-      return dashboardData.climateData;
+
+    // Gey data.
```
Comment (nerdstein) [inline examples/customDataHandlers.js]: spelling typo

File: examples/customDataHandlers.js [hunk 2]
```diff
@@ -85,13 +85,51 @@ let customDataHandlers = {
...
+    const data = dashboardData.climateData || [];
```
Comment (nerdstein) [inline examples/customDataHandlers.js]: "climateData" seems a bit odd here... is this a specific implementation?

Comment (nerdstein) [reply examples/customDataHandlers.js]: it's also not a parameter

Comment (alexiscott) [reply examples/customDataHandlers.js]: It's the data that has been used for all of the demos, so it's common to the example page.

**Pull request merged**